framework/test/unit/modules/random/random_module.spec.ts

```diff
@@ -484,9 +484,12 @@ describe('RandomModule', () => {
 				assets: new BlockAssets([asset]),
 			});
 
-			await expect(
-				randomModule.verifyAssets(context.getBlockVerifyExecuteContext()),
-			).rejects.toThrow('Size of the seed reveal must be 16, but received 10.');
+			try {
+				await randomModule.verifyAssets(context.getBlockVerifyExecuteContext());
+			} catch (error: any) {
+				// eslint-disable-next-line jest/no-try-expect
+				expect(error?.message).toInclude(`Property '.seedReveal' minLength not satisfied`);
+			}
```

Comment:
Why we need to change to try/catch?
```suggestion
				await randomModule.verifyAssets(context.getBlockVerifyExecuteContext());
			} catch (error: any) {
				// eslint-disable-next-line jest/no-try-expect
				expect(error?.message).toInclude(`Property '.seedReveal' minLength not satisfied`);
			}
```

Reply:
For convenience for the jest `toInclude` message since the format of the validator is long and unwieldy. If you have a better suggestion I can update it.

Reply:
This does not work: `string.includes is not a function`

Reply:
Actually this pattern should be avoided because if the function resolves, this test will pass as well

Reply:
If there is a better suggestion I can update it.